### PR TITLE
[ICDS Dashboard]New Ls launched definition

### PR DESCRIPTION
**Pull request by Rohit25negi**
Creating a Ucr for tracking the Ls launched. 
The definition is basically. If any of the forms (added in the ucr) is submitted then the LS is considered as launched.

fyi, reviewers, this can go after this https://github.com/dimagi/commcare-hq/pull/25849/files which is of higher priority.

### CODE REVIEW

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+          "location_property": "_id",
+          "type": "ancestor_location"
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "state_id",
+        "type": "expression"
+      },
+     {
+        "type": "count",
```
Comment (calellowitz) [inline custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: what is this?

**Review by calellowitz — COMMENTED** [review]
Pending that comment, looks fine

**Comment by Rohit25negi** [comment]
@calellowitz  replied.

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: The value of this column will be 1 for each row.

Comment (Rohit25negi) [reply custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: I removed it. I realized It was just used in the previous ucrs unnecessarily

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        }
+      }
+    },
+    "named_filters": {},
+    "engine_id": "icds-ucr-citus",
+    "disable_destructive_rebuild": true,
+    "sql_settings": {
+      "citus_config": {
+        "distribution_type": "hash",
+        "distribution_column": "location_id"
```
Comment (Rohit25negi) [inline custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: This distributed ucr. Is anything special needs to do for this. Or not any other change is required.


Or you think this does not need to be distributed.
 @calellowitz

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json (hunk 1 above)
Comment (calellowitz) [reply custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: any reason to use this over just `constant`

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    ],
+    "configured_indicators": [
+      {
+        "display_name": "Supervisor ID",
+        "datatype": "string",
+        "type": "expression",
+        "transform": {},
+        "is_nullable": false,
+        "is_primary_key": true,
+        "column_id": "location_id",
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: I think this should be `supervisor_id`. If we want to distribute this on Citus, using supervisor_id will ensure it is local to other data about the supervisor

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+              "form",
+              "meta",
+              "timeEnd"
+            ]
+          },
+          "type": "root_doc"
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "submitted_on",
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: should this be `timeend`? I think that other UCRs use `submitted_on`, but I generally think that's confusing as in other parts of HQ "submission time" is received_on and "completion time" is timeend

Comment (Rohit25negi) [reply custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: That make sense. I can change it to timeend

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json (hunk 3 above)
Comment (Rohit25negi) [reply custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: @emord is that needed even when ` "distribution_column": "location_id"`?

File: custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json (hunk 2 above)
Comment (calellowitz) [reply custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: Looks fine, and distributing it is a good idea. but i would change the column name to `supervisor_id` if for no other reason than consistency

Comment (emord) [reply custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json]: Moving distribution discussion to the same thread. From my other question:

> I think this should be supervisor_id. If we want to distribute this on Citus, using supervisor_id will ensure it is local to other data about the supervisor

@calellowitz isn't the change to `supervisor_id` needed to ensure data about a supervisor is located on the same shard? I recall you mentioning that when you use the same column name, citus distributes it in the same way. So the column name change is not just for consistency

**Review by calellowitz — APPROVED** [review]

File: custom/icds_reports/tasks.py [hunk 5]
```diff
@@ -166,6 +166,7 @@
     {'type': 'child_list', 'name': 'static-child_health_cases'},
     {'type': 'ccs_record_list', 'name': 'static-ccs_record_cases'},
     {'type': 'ls_vhnd', 'name': 'static-ls_vhnd_form'},
+    {'type': 'ls_usage','name':'static-ls_usage_forms'},
```
Comment (emord) [inline custom/icds_reports/tasks.py]: Is this still used? for some reason I thought we could kill this mapping

**Review by emord — APPROVED** [review]

**Pull request merged**